.github/ISSUE_TEMPLATE/alpha_feature.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Alpha Feature Requirements
+about: Requirements for promoting a feature to alpha
+---
+
+This page lists the requirements for promoting a feature to alpha. Please check off and document the steps as they are completed.
+
+**Feature Name:** 
+--- 
+
+### Requirements: 
+
+**Design**
+
+- [ ] Requires approved RFC in order to merge. 
+
+**Config**
+
+- [ ] Requires explicit user action to enable (e.g. a config field, config resource, or installation action). 
+
+**Docs**
+
+- [ ] Reference docs are published to istio.io. 
+- [ ] Basic feature docs are published on istio.io describing what the feature does, how to use it, and any caveats. 
+- [ ] Whenever possible, documentation has automated istio.io tests written. 
+- [ ] A reference to the design doc/issue is published on istio.io. 
+- [ ] Release notes entries added as appropriate
+- [ ] Upgrade notes entries added as appropriate
+
+**Tests**
+
+- [ ] Test coverage is >= 70%
```

Comment:
given we don't measure this for anything today, seems hard to enforce

Reply:
agreed.

Reply:
Do we want to add tooling for this? These code coverage levels are a requirement we've had for promotion of features for the last couple of years: https://github.com/istio/community/blob/master/FEATURE-LIFECYCLE-CHECKLIST.md. I think having a level of coverage is a good thing but agree that we currently don't have any way to check it. We could always start checking on new code.

Reply:
Actually, I'm not sure that we can start checking on new code. We could do it on a PR basis, but that misses any previous promotions. We could do it on a release basis, but previous promotions are in a previous release.... Still, might not hurt to start looking at our test coverage.